app/models/user.rb

```diff
@@ -82,7 +82,7 @@ class User < ActiveRecord::Base
 
   def before_add_method(role)
     return if !Role.admins.include?(role.name) || has_role?(:panitia)
-    raise 'User is not panitia!'
+    raise 'User is not panitia! Run `User.find(your_id).add_role :panitia` first.'
```

Comment:
Why is this here?

Might it be that you have not rebased from master? Basically, you did two commits in your past PR ("add solution if you're not panitia" and "Add suggested action to be panitia before being admin"), but they are squashed into one commit when the previous PR is merged into master.

So right now the git tree is like this:
```
X - Y (master)
|
Y1 - Y2 - Z1 - Z2 - Z3 - Z4 (your branch)
```

Previously, when working on the panitia stuff you added the commits Y1 and Y2. They are combined into one commit Y in master. However, looking at the commit history of this branch (https://github.com/ktomatematika/kontes-terbuka/pull/126/commits) it seems like you still have the two commits Y1 and Y2.

Now we want the git tree to be like this
```
X - Y (master)
    |
    Z1 - Z2 - Z3 - Z4 (your branch)
```

To achieve this, we want to rebase from master. Rebase simply means, we want to base your branch into a different commit: from commit X as base, to commit Y as base. However, we also want to remove these two commits Y1 and Y2.

To do this, you want to use interactive rebase. You can do so from your branch, by running `git rebase -i master` (where -i stands for interactive). When you run this command, you will see a text editor pop up, with a line describing each commit, preceded by the word "pick". You can either replace the word "pick" by "drop" on these two commits Y1 and Y2, or you can just delete these two lines. By doing so, you will be left with just the 4 commits Z1 to Z4.

Do ping me if you feel that this is unclear :)

Reply:
I found problem :
fatal: invalid upstream 'master'
when i was googling, solution from stacoverflow is :
git rebase -i origin/master
I'm not sure

Reply:
Oh sorry our main branch is called `production`. So it should be `git rebase -i production` instead.